### DISCUSSION TITLE
SIDKick for clumsy fingers

### DISCUSSION THREAD

**Discussion by tobozo** [frntc/Sidekick64#49] [Q&A] [answered]
hi, thanks for this great project !

I'm unable to solder surface mount components and would like to have this assembled by the pcb manufacturer.

However doing this requires a specifically formatted BOM list along with a CPL file for the pick and place machine.

Can you share those files (provided they exist) or eventually share the whole kicad SIDKick project, lib and schematics so it becomes possible to produce said BOM list and CPL file in addition to the existing gerber files?

The `SIDKick v04s` board is what I'm after but this would certainly be useful to everyone if this was available for all versions.

**Comment by frntc** [comment, marked as answer]
Hi, 

I know what you mean ;-) The 0.4-boards are even more painful to assemble than the 0.3-boards. I understand your point very well, but I'm not yet sure if I want BOM and CPL files to be included in the repository. But there is one solution: you could contact km@restore-store.de -- he has populated PCBs available.

**Reply by tobozo** [reply]
Thanks to your answer I had a reply from the email address you provided :partying_face: however I got a bit confused about the longboard/shortboard versions, for some reason I thought the v04/v04s would fit both, so I'll be ordering both.

I noticed you turned off the issue tracker on the SIDKick repository, does it mean the project is archived and won't receive any update? 
The SD card on the Teensy 4.1 doesn't seem to be used by the firmware and I'm wondering if there's a possible feature opportunity I can research on.

**Comment by tobozo** [comment]
thanks!

I tried the website and could only find some SideKick64 boards, but no SIDKick were listed, let's hope the email works better :crossed_fingers: 

![image](https://user-images.githubusercontent.com/1893754/201798124-e75c4460-0493-4cca-bb10-a213b93c7a13.png)

**Comment by frntc** [comment]
That's why I suggested writing a mail, which (I was told) did indeed work ;-)

**Reply by tobozo** [reply]
no worries I'll mark the post as answered as soon as I the order is complete and delivered

**Reply by tobozo** [reply]
came with a few treats :p 
![image](https://user-images.githubusercontent.com/1893754/203321460-50250625-03c9-4805-8e2d-301757322204.png)

**Reply by tobozo** [reply]
hey :-)

After killing two v04s units I have some feedback suggestions:

An accidental SID socket inversion "test" was performed, the SwinSID survided but the SIDKick v04s didn't.

I totally blame myself on this, especially for SwinSID since it has a half-moon mark, but this made me think of a possible improvement for the SIDKick.

    Pay attention to correctly orient and insert the Teensy and the 
    SIDKick (see backside of PCB for markings) 

If you're planning on updating the design, please consider adding a half-moon to both silkscreens to reduce probability of plugging the SIDKick upside-down in the SID socket, most SID clones have it. Adding back the Teensy pin0 silkscreen hint would also make the installation less of a treasure hunt.

If no design update is planned for the SIDKick, adding a small sticker, or carving the pcb with a lime tool achieves the same effect, and can be applied to existing kits without requiring a redesign.

**Reply by frntc** [reply]
oh no! Are you sure they are killed? I remember someone inserting them in a C128D in the wrong direction, and it survived.

adding something to the silkscreen is a good idea (when/in case there will a revision), I always have to look at picture to correctly insert the Teensy and the SIDKick :-)

**Reply by tobozo** [reply]
yes, the 12v pin also killed 2x lolin d32pro on the other side of the circuit :facepalm: 

good thing is this gave me motivation to play with kicad :+1:
